src/db/config.ts

```diff
@@ -17,7 +17,7 @@ export const getDBConfig = (configService: ConfigService) => {
     cli: { migrationsDir: 'src/migrations' },
     migrationsRun: true,
     migrationsTableName: 'migrations_iam_cache_server',
-    logging: false,
+    logging: ['error', 'migration', 'warn', 'info'],
```

Comment:
It would be better to have this as config so that we can control per environment
```suggestion
    logging: JSON.parse(configService.get<string>('TYPEORM_LOGGING')),
```